src/__tests__/random.test.js

```diff
@@ -0,0 +1,73 @@
+import * as random from '../random';
+import validate from 'uuid-validate';
+
+describe('random', () => {
+  it('Generates random boolean values', () => {
+    let hasTrue = false;
+    let hasFalse = false;
+    for (let i = 0; i < 10; i++) {
+      if (random.getBoolean()) {
+        hasTrue = true;
+      } else {
+        hasFalse = true;
+      }
+    }
+    expect(hasTrue).toBeTruthy();
```

Comment:
I don't enough about how `Math.random` works under the hood but my hunch is that these assertions might fail from time to time. Can we instead use a `while` loop to keep generating under random values until all our assertions pass?
```
while (let i = 0; i++) {
  let booValue = random.getBoolean();
  expect(typeof(true)).toBe('boolean');
  if (booValue === true) {
     hasTrue = true;
  }
  if (booValue === false) {
     hasFalse = true;
  }
  if (hasTrue and hasFalse) {
    return;
  }
  if (i > 10000) {
    fail('Didn't generate all expected values after 10000 calls');
  }
}
```